eclipse/plugins/org.switchyard.tools.ui.editor/src/org/switchyard/tools/ui/editor/impl/MultiPageEditor.java

```diff
@@ -902,6 +905,12 @@ public void showList(CTabFolderEvent event) {
         createDesignEditor();
         createDomainPage();
         createSourceViewer();
+
+        // when editor is opened, count the first time as a usage event
+        UsageEventType eventType = new UsageEventType(Activator.getDefault(), 
+                Activator.PLUGIN_ID + ";editor-startup"); //$NON-NLS-1$
```

Comment:
You also need to register the event type before using it.
We need to register it because we have to show all registered event types in preferences so users always can check what we track. Unregistered event type won't send anything to GA.
See https://github.com/jbosstools/jbosstools-devdoc/blob/master/api/usage/usage_api.adoc#event-registration
And an example in servers: https://github.com/jbosstools/jbosstools-server/blob/jbosstools-4.2.0.Beta1x/as/plugins/org.jboss.ide.eclipse.as.ui/jbossui/org/jboss/ide/eclipse/as/ui/JBossServerUIPlugin.java#L95-L96

And also, the best plase to create and register an event type is in your plugin start method. You don't need to do it every time you want to send a new event. You register an event type once then use it as many times as you want when sending events.